css/selectors/nesting.json

```diff
@@ -28,9 +28,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": "16.5",
-              "partial_implementation": true,
-              "notes": "Does not support nested rules that start with a type selector."
+              "version_added": "17.2"
```

Comment:
shouldn't we keep the entry about partial implementation in 16.5 and add a new entry for the full support in 17.2 ?

Reply:
Yeah, you're right. I have corrected it.